canonical/canonicalwebteam.templateparser-backend
# Added sso authentication to the endpoints

## Done
- Added sso authentication, with default settings for `canonical-web-monkeys`

## QA
- Clone this branch and open `/get-tree/canonical.com`
- You should be redirected to ubuntu sso, asked to log in, and redirected back if successful

Fixes [WD-8291](https://warthogs.atlassian.net/browse/WD-8291?atlOrigin=eyJpIjoiNjNjMGU2ZGZlYWU2NDE2NGEyNmM5N2U2NzVlODk4NTkiLCJwIjoiaiJ9)

[WD-8291]: https://warthogs.atlassian.net/browse/WD-8291?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
